README.md

```diff
@@ -1,17 +1,17 @@
 <h1 align="center">
-<b>Olas Modius Quickstart</b>
+<b>Optimus Quickstart</b>
 </h1>
 
 
 
-## Olas Modius Agent: Functionality Overview
+## Optimus Agent: Functionality Overview
 
-Olas Modius is an autonomous trader that operates in the crypto echosystem on users' behalf. It requires minimum intervention by you as an end user--it is enough that you fund the agent with initial investment assets and let it run. Modius will intelligentely invest crypto assets in DEXs on your behalf and grow your portfolio.
+Optimus is an autonomous trader that operates in the crypto echosystem on users' behalf. It requires minimum intervention by you as an end user--it is enough that you fund the agent with initial investment assets and let it run. Optimus will intelligentely invest crypto assets in DEXs on your behalf and grow your portfolio.

```

Comment:
Optimus is an autonomous trader that operates in the crypto ecosystem on users' behalf. It requires minimum intervention by you as an end user--it is enough that you fund the agent with initial investment assets and let it run. Optimus will intelligently invest crypto assets in DEXs and grow your portfolio.
```suggestion
Optimus is an autonomous trader that operates in the crypto echosystem on users' behalf. It requires minimum intervention by you as an end user--it is enough that you fund the agent with initial investment assets and let it run. Optimus will intelligentely invest crypto assets in DEXs on your behalf and grow your portfolio.
```
```suggestion
Optimus is an autonomous trader that operates in the crypto echosystem on users' behalf. It requires minimum intervention by you as an end user--it is enough that you fund the agent with initial investment assets and let it run. Optimus will intelligentely invest crypto assets in DEXs on your behalf and grow your portfolio.

```